content/applications/sales/point_of_sale/restaurant/tips.rst

```diff
@@ -1,31 +1,88 @@
-===================================
-Integrate a tip option into payment
-===================================
+====
+Tips
+====
 
-As it is customary to tip in many countries all over the world, it is
-important to have the option in your PoS interface.
+Tipping is customary in multiple countries. Point of Sale allows tipping in :doc:`shops
+<../overview/getting_started>`, :doc:`bars <../restaurant>`, or :doc:`restaurants <../restaurant>`.
 
-Configure Tipping
-=================
+.. _configuration:
 
-To activate the *Tips* feature, go to :menuselection:`Point of Sale
---> Configuration --> Point of sale` and select your PoS.
+Configuration
+=============
 
-Under the Bills & Receipts category, you will find *Tips*. Select it
-and create a *Tip Product* such as *Tips* in this case.
+To allow tipping in your POS, activate the :guilabel:`Tips` feature in :menuselection:`Point of Sale
+--> Configuration --> Settings`. At the top of the page, select the POS in which you wish to allow
+**tipping**, scroll down to the :guilabel:`Payment` section and check :guilabel:`Tips`. Once
+enabled, add a :guilabel:`Tip Product` in the corresponding field, and save. The designated product
+will be used as a reference on customers' receipts.
 
```

Comment:
Suggestion:
Describe here that they can also enable adyen and tipping after payment, with custom links.

This way you leave the configuration in this section, and can explain only how it works in the other section.